lib/node_modules/@stdlib/math/base/complex/exp/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@stdlib/math/base/complex/cis",
+  "version": "0.0.0",
+  "description": "Compute the cis function of a complex number.",
```

Comment:
> Compute the exponential function of a complex number.

Also, compute the exponential function OF a complex number, or FOR a complex number?

Reply:
I opted for "of" a complex number for consistency, though I'm glad to change that. Some take a form more like "[Natural logarithm of the Beta function](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/math/base/special/betaln)" while most others start with "Compute". No strong feelings on my part.

Reply:
That is fine with me.